WriteVHDLSyntax.py

```diff
@@ -646,10 +646,15 @@ def writeTopLevelMemoryType(mtypeB, memList, memInfo, extraports, delay = 0, spl
                     merge_parameterlist += "        NUM_INPUTS => "+str(numInputs)+",\n"
                     merge_parameterlist += "        NUM_EXTRA_BITS => 2,\n"
                     merge_portlist += "        bx_in => TP_bx_out,\n"
+                    merge_portlist += "        bx_in_vld => TP_bx_out_vld,\n"
                     merge_portlist += "        rst => '0',\n"
                     merge_portlist += "        clk => clk,\n"
                     merge_portlist += "        enb_arr => open,\n"
-                    merge_portlist += "        bx_out => open,\n"
+                    #This will make output for first stream_merge module (is there a less hacky way?)
+                    if (seed+PCGroup)=='L1L2ABC': 
+                      merge_portlist += "        bx_out => TP_bx_out_merged,\n"
+                    else:
+                      merge_portlist += "        bx_out => open,\n"
```

Comment:
@mcoshiro Does `TP_bx_out_merged` have to come from `MERGE_STREAM_TPARL1L2ABC` specifically? If it can just come from the first `tf_merge_streamer`, then maybe using a boolean would be less hacky, e.g.:
```python
first_merge_streamer = True
…
    if first_merge_streamer:
      merge_portlist += "        bx_out => TP_bx_out_merged,\n"
      first_merge_streamer = False
    else:
      merge_portlist += "        bx_out => open,\n"
```